src/promptflow/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 ### Bugs Fixed
 
 - [SDK/CLI] Keep original format in run output.jsonl.
+- [Executor] Fix failure when node refers bypassed node.
```

Comment:
When aggregation node refers bypassed node

Reply:
All the changes have been moved to https://github.com/microsoft/promptflow/compare/main...release/promptflow/1.0.0